velox/exec/tests/PrintPlanWithStatsTest.cpp

```diff
@@ -154,7 +154,6 @@ TEST_F(PrintPlanWithStatsTest, innerJoinWithTableScan) {
       "          numRamRead                sum: 0, count: 1, min: 0, max: 0\n"
       "          numStorageRead            sum: .+, count: 1, min: .+, max: .+\n"
       "          prefetchBytes             sum: .+, count: 1, min: .+, max: .+\n"
-      "          queuedWallNanos           sum: .+, count: 1, min: .+, max: .+\n"
```

Comment:
@mbasmanova why did `queuedWallNanos` go away?

Reply:
TableScan operator used to incorrect report itself blocked at the start of execution. This was due to blockingFuture_ variable being initialized to a valid completed future. Now, it is initialized to an invalid future (ContinueFuture::makeEmpty()). TableScan is not blocked at the start of execution and doesn't accumulate queuedWallNanos.